eventsources/sources/amqp/start.go

```diff
@@ -19,6 +19,7 @@ package amqp
 import (
 	"context"
 	"encoding/json"
+	"github.com/ghodss/yaml"
```

Comment:
Please use `sigs.k8s.io/yaml` which is a better implementation forked from `ghodss` one. The `ghodss` one has some issues.